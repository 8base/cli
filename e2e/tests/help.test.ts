import { runner } from '../runner';

it('As a user, I can use help flag for see help information about commands.', async () => {
  const { stdout } = await runner()('--help');

  expect(stdout).toMatchSnapshot();
});