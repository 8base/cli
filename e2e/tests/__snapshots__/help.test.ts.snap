// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`As a user, I can use help flag for see help information about commands. 1`] = `
"Usage: 8base <command> [OPTIONS]

Commands:
  8base config        Select workspace
  8base deploy        Deploy project
  8base describe      Describe project functions and their types
  8base export        Export current workspace data schema
  8base import        Import schema file to the current workspace
  8base init          Initialize project
  8base invoke        Invoke deployed function remotely
  8base invoke-local  Invoke function locally
  8base login         Login via browser
  8base logout        Clears local login credentials and invalidates API session
  8base logs          View function logs
  8base package       Package application without deploying

Options:
  -h, --help     Show help                                             [boolean]
  -v, --version  Show version number                                   [boolean]"
`;
